src/iop/exposure.c

```diff
@@ -87,6 +87,7 @@ typedef struct dt_iop_exposure_gui_data_t
   GtkLabel *deflicker_used_EC;
   GtkWidget *compensate_exposure_bias;
   GtkWidget *compensate_hilite_preserv;
+  volatile float effective_exposure; // used to cache the final computed exposure
```

Comment:
No idea if `volatile` is really needed here -- could this be read/written in different threads without some synchronization mechanism guaranteeing visibility?

Reply:
I don't understand. A gui_data is used in gui threads only if I'm not mistaken and there is a single gui thread. So volatile seems superfluous to me. In any case don't think volatile on a float is sufficient to ensure atomic access.

Reply:
Removed, thanks.